master/buildbot/www/hooks/github.py

```diff
@@ -176,7 +176,8 @@ def get_changes(self, category, payload):
         elif category == 'release':
             branch = payload['release']['tag_name']
 
-        log.msg("Processing %s #%s" % (category, branch))
+        log.msg('Processing {0},{1}'.format(category, branch),
```

Comment:
if you want to implement this with log level, please use new logger api:

https://twistedmatrix.com/documents/current/core/howto/logger.html
```python
from buildbot.util.logger import Logger

log = Logger()
...
log.debug("xxx")
```